_posts/2015/2015-11-24-essence-of-tasks.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "On the essence of the task"
+date: "2015-11-24 07:23:00"
+icon: tasks
+categories: tech-stories
+authors: ["Timo H."]
+---
+
+{% highlight text %}
+"Task = Stuff needs to be done, do it, and you are rid of it!"
+{% endhighlight %}
+
+The problem with tasks is however, that there are quite many of them.
+Infinite, to be exact.
+Some mathematician could say which infinite, but pretty much anyway.
+
+The point is that they cannot be all done, definitely not at once.
+Some of them are actually preventing that others of them can be done.
+Even the amount of the tasks for some limited subarea, for example for bringing the best e-commerce to small people who could not manage it alone, is in terms of human lifespan infinite.
+And next subsection of it, for example perfect customer satisfaction, likewise.
+
+You need some kind of a system, to find out which tasks are the most important, which are best for reaching the goal, and those you do first.
+
+Within a workshop we discussed four of them:
+
+* [Getting things done ](https://en.wikipedia.org/wiki/Getting_Things_Done)
+* [Todo-Lists](https://en.wikipedia.org/wiki/Time_management#Task_list_organization)
+* [Not-Todo Lists](https://www.pinterest.com/pin/76772368624218376/)
+* [Kanban](https://en.wikipedia.org/wiki/Kanban)
+
+The bottom line of this workshop was to pick one system, which suits your role and your tasks best.
+Without a correct system one ends up working chaotically, which on long term leads to working on wrong tasks, wasting own and other peoples time and missing the target.
```

Comment:
Without a correct system one ends up working chaotically which, in the long term, leads to working on the wrong tasks, wasting your own and other people's time and ultimately missing targets.